misc/jenkins/compile_other_versions/prepare_bundle.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
+BUNDLE_NAME="$BUNDLE_NAME"
```

Comment:
BUNDLE_NAME="$BUNDLE_NAME"?

Reply:
Search and replace got me. I even thought of that as I was doing it but somehow still got it wrong.